src/elements/resource.js

```diff
@@ -0,0 +1,38 @@
+export default function(namespace) {
+  const ArrayElement = namespace.getElementClass('array');
+
+  class Resource extends ArrayElement {
+    constructor() {
+      super(...arguments);
+
+      this.element = 'resource';
+      this._attributeElementKeys = ['hrefVariables'];
+    }
+
+    get href() {
+      return this.attributes.getValue('href');
+    }
+
+    set href(value) {
+      this.attributes.set('href', value);
+    }
+
+    get hrefVariables() {
+      return this.attributes.get('hrefVariables');
+    }
+
+    set hrefVariables(value) {
+      this.attributes.set('hrefVariables', value);
+    }
+
+    get transitions() {
```

Comment:
If you are renaming `transitions` at [this](https://github.com/refractproject/minim-api-description/pull/1/files#r38706955) place, please rename it here too.